src/parser/mod.rs

```diff
@@ -8888,6 +8888,64 @@ impl<'a> Parser<'a> {
         }
     }
 
+    pub fn parse_table_index_hints(&mut self) -> Result<Vec<TableIndexHints>, ParserError> {
```

Comment:
```suggestion
    fn parse_table_index_hints(&mut self) -> Result<Vec<TableIndexHints>, ParserError> {
```
I wonder if we need to have this function public, if we do feel free to leave as is and we can also include a doc string describing its behavior in that case?